### DISCUSSION TITLE
How to Use Functions?

### DISCUSSION THREAD

**Discussion by igorpanin** [n3d1117/chatgpt-telegram-bot#396] [Q&A] [answered]
Thank you for 0.3.0. 

I really want to use functions, but I'm uncertain about how to do so.

OPENAI_MODEL=gpt-4
MAX_TOKENS=2400

After upgrading the code to version 0.3.0, the only modification I made was adding these lines in the .env file:
<img width="343" alt="Screenshot 2023-08-04 at 23 02 59" src="https://github.com/n3d1117/chatgpt-telegram-bot/assets/125222668/943bda1e-2623-4141-8701-7795568b4592">

However, even after restarting the code and the bot, I continue to receive the same old responses:
<img width="527" alt="Screenshot 2023-08-04 at 23 00 02" src="https://github.com/n3d1117/chatgpt-telegram-bot/assets/125222668/d1188fc4-79fd-42dc-81ec-6c37cc441f97">

Could you please guide me on what else needs to be done apart from adding the lines mentioned above to the .env file?

(I understand that if there are more than just a few more steps involved, it might be best for me to seek assistance from someone with coding experience instead of attempting it myself)

Thank you.

**Comment by noriellecruz** [comment, marked as answer]
did you not remove the # from the lines?

**Reply by igorpanin** [reply]
Wow, thank you very much!

**Reply by n3d1117** [reply]
Glad it's solved! Thanks @noriellecruz
